apps/barista-design-system/src/pages/overview-page/components/component-preview-tile/component-preview-tile.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../../../../libs/barista-components/core/src/style/colors';
+@import '../../../../styles/global/variables';
+
+$tilesize: 290px;
+$tilefontsize: 28px;
+$tilefontsize-big: 42px;
+$gridgap: 20px;
+$imagesize: 156px;
+
+@mixin tile-theme($link-color, $hover-color, $active-color) {
+  color: $link-color;
+
+  :hover,
+  &:hover {
+    border-color: $hover-color;
+    text-decoration: none;
+
+    &.ba-tile-image-preview {
+      border-bottom: 1px solid #ddd;
+    }
+  }
+
+  :focus,
+  &:focus {
+    outline: 2px solid $hover-color;
+  }
+
+  :active,
+  &:active {
+    border-color: $active-color;
+  }
+
+  .ba-tile-image {
+    color: $hover-color;
+  }
+
+  .ba-tile-badge-favorite,
+  &.ba-tile-badge-favorite {
```

Comment:
This defines the styles for the element with the class `ba-tile-badge-favorite`

Reply:
yeah, you definitely have to keep the first selector (line 37), I just meant the second one with the parent selector, should have clarified that. for this one it is pretty much useless, since the parent only ever has the class `ba-component-preview-tile`, or is there a case where it also has `.ba-tile-badge-favorite`?